test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2825,7 +2916,7 @@ public void testExtraMetadata_docAndExtra() throws Exception {
     configEntries.put("db.singleDocContentSql",
         "select id, name from data where id = ?");
     configEntries.put("db.modeOfOperation", "rowToText");
-    configEntries.put("db.metadataColumns", "name");
+    configEntries.put("db.includeAllColumnsAsMetadata", "true");
```

Comment:
Why make this change?

Reply:
Changing this setting adds coverage for the case you noted above for line 694.